src/api/SFA.DAS.Provider.Events.Infrastructure/Data/DcfsPaymentRepository.cs

```diff
@@ -77,30 +78,44 @@ public class DcfsPaymentRepository : DcfsRepository, IPaymentRepository
                     childCollection: p => p.PaymentsDueEarningEntities
                 ).ConfigureAwait(false);
 
-                var pagedResults =  PageResults(result, page, pageSize);
+                var pagedResults = PageResults(result, page, pageSize);
                 return pagedResults;
             }
         }
 
+        public async Task<PaymentStatistics> GetStatistics()
+        {
+            using (var connection = await GetOpenConnection().ConfigureAwait(false))
+            {
+                var sql = "  SELECT  (SELECT count(1) " +
+                          "  FROM Payments.Payments p" +
+                          "  INNER JOIN PaymentsDue.RequiredPayments rp ON p.RequiredPaymentId = rp.Id) as TotalNumberOfRecievedPayments," +
+                          "  (SELECT count(1) " +
+                          "  FROM Payments.Payments) as TotalNumberOfPayments";
+

```

Comment:
this can be done with a single query with left join

Reply:
yep, change made